cmd/nodepool/core/create.go

```diff
@@ -29,13 +31,23 @@ type PlatformOptions interface {
 	Type() hyperv1.PlatformType
 }
 
+func (o *CreateNodePoolOptions) CreateExecFunc(platformOpts PlatformOptions) func(cmd *cobra.Command, args []string) error {
```

Comment:
CreateRunFunc?
https://github.com/spf13/cobra/blob/e04ec725508c760e70263b031e5697c232d5c3fa/command.go#L98

Reply:
I just moved existing (duplicated) code from the platforms files and wrote it once, to be used from all platforms
What is the exacly issue here?

Reply:
I'm just suggesting to use a single term in naming and replace "exec" by "run" as it seems to be how the owning package refers to them https://github.com/spf13/cobra/blob/e04ec725508c760e70263b031e5697c232d5c3fa/command.go#L98
`cmd.RunE`

Reply:
Done